library/src/main/scala/com/monsanto/arch/kamon/prometheus/PrometheusExtension.scala

```diff
@@ -43,6 +50,13 @@ class PrometheusExtension(system: ExtendedActorSystem) extends Kamon.Extension {
   /** The Spray endpoint. */
   val route: Route = endpoint.route
 
+  /** For internal use (healthchecks, other routes, etc.) */
+  def requestSnapshot() = Option(snapshot.get())
+
+  def flushBuffer() = {
+    buffer ! FlushBuffer
+  }
+
```

Comment:
I noticed that `flushBuffer` is used in testing.  Did you have any other uses in mind?  Also, what about `requestSnapshot`?  It's not used within the library, so is it something that is useful for your application?